### PR TITLE
source-holiday-api contribution from natikgadzhi

### PR DESCRIPTION
**Pull request by natikgadzhi**
Auto-generated PR by the Connector Builder for source-holiday-api

### CODE REVIEW

File: airbyte-integrations/connectors/source-holiday-api/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Holiday API
+This directory contains the manifest-only connector for `source-holiday-api`.
+
+
+## Usage
+
+- You can use this connector as any other connector in Airbyte Marketplace.
+- You can load this connector in `pyairbyte` usinng `get_source`!
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-holiday-api/README.md:8]: @bnchrch a little typo in readme template

```suggestion
- You can load this connector in `pyairbyte` using `get_source`!
```

File: airbyte-integrations/connectors/source-holiday-api/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Holiday API
+This directory contains the manifest-only connector for `source-holiday-api`.
+
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-holiday-api/README.md:3]: Extra line
```suggestion
```

**Review by natikgadzhi — COMMENTED** [review]
Formatting still doesn't pass for the manifest — Do we do this on frontend later?

File: airbyte-integrations/connectors/source-holiday-api/README.md (hunk 1 above)
Comment (bnchrch) [reply airbyte-integrations/connectors/source-holiday-api/README.md:8]: Fixed: https://github.com/airbytehq/airbyte-platform-internal/pull/13505

**Pull request closed without merging**